cityofaustin/atd-vz-data
# Implement soft deletes for crash notes

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/18788

Users cant delete recommendations only edit, so we dont need to implement soft-deletes for them.

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Check out this branch, start your local db, apply the migrations and metadata.
2. Go to a crash, add a note and delete your note. Make sure you can still find the note in the `crash_notes` table but the is_deleted column will say true.
3. Run the down migrations


---
#### Ship list
